bindings/pyroot_experimental/PyROOT/test/tcomplex_radd_rsub.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import ROOT
+from ROOT import TComplex
+
+class TestTComplexMethods(unittest.TestCase):
+        """
+        Test for the operators of TComplex:
+        __add__, __sub__, __radd__, __rsub__.
+        """
+
+        c1 = TComplex(4.,0)
```

Comment:
Perhaps you do not need it here, but just FYI, you can have setup methods in unittests, e.g.:
https://github.com/root-project/root/blob/master/bindings/pyroot_experimental/PyROOT/test/tdirectory_attrsyntax.py#L17

So you can encapsulate some more complex behaviour that only happens once (when loading the test class).